server/controllers/recommendation/ppcs/indeterminateSentence/edit/editOffenceController.ts

```diff
@@ -0,0 +1,102 @@
+import { NextFunction, Request, Response } from 'express'
+import { ppudReferenceList, updateRecommendation } from '../../../../../data/makeDecisionApiClient'
+import { makeErrorObject } from '../../../../../utils/errors'
+import { strings } from '../../../../../textStrings/en'
+import { nextPageLinkUrl } from '../../../../recommendations/helpers/urls'
+import { isDefined, isEmptyStringOrWhitespace } from '../../../../../utils/utils'
+import { RecommendationResponse } from '../../../../../@types/make-recall-decision-api'
+import { NamedFormError } from '../../../../../@types/pagesForms'
+
+async function get(req: Request, res: Response, next: NextFunction) {
+  const {
+    recommendation,
+    unsavedValues,
+    user: { token },
+  } = res.locals
+
+  const recommendationResponse = recommendation as RecommendationResponse
+  const ppudSentence = recommendationResponse.ppudOffender.sentences.find(
+    s => s.id === recommendation.bookRecallToPpud.ppudSentenceId
+  )
+  const existingOffenceDescription = ppudSentence.offence.indexOffence
+  const pendingSentenceData = recommendationResponse.bookRecallToPpud.ppudIndeterminateSentenceData
+
+  // A list of all index offences for the PoP
+  const list = await ppudReferenceList(token, 'index-offences')
+  const allOffences = list.values.map(value => {
+    return {
+      text: value,
+      value,
+    }
+  })
+
+  res.locals = {
+    ...res.locals,
+    page: {
+      id: 'editOffence',
+    },
+    existingOffenceDescription,
+    // If the value hasn't changed, don't preselect anything in the autocomplete
```

Comment:
Might this not be confusing if they've set it to the same as the existing one, and then go back to add a comment but have to enter the offence again?

Reply:
Isn't the point of this to edit the values if the information on PPUD is incorrect - why would they select the same offence in that case?

Reply:
Aha, that makes sense. I thought this was one of the fields that isn't pre-populated, but I was misremembering (should've just checked, sorry!).